src/Tools/Md2Html/Md2HtmlSanitizer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Ganss.XSS;
+
+namespace Md2Html
+{
+    /// <summary>
+    /// HtmlSanitizer class that tracks sanitizing changes
+    /// </summary>
+    internal class Md2HtmlSanitizer : HtmlSanitizer
+    {
+        /// <summary>
+        /// Is True if any content needed to be sanitized.
+        /// </summary>
+        internal bool Changed { get; set; }
+
+        internal Md2HtmlSanitizer()
+        {
+            RemovingAtRule += ChangedEvent;
+            RemovingAttribute += ChangedEvent;
+            RemovingComment += ChangedEvent;
+            RemovingCssClass += ChangedEvent;
+            RemovingStyle += ChangedEvent;
+            RemovingTag += ChangedEvent;
+        }
+
+        ~Md2HtmlSanitizer()
```

Comment:
do you think this is better than disposal?

Reply:
I am still learning about c# destructors/finalizers and IDisposal. Looks like recommendation is to never remove event handlers in destructors as it only executes at time for garbage collect if the object has a finalizer. Implement iDisposable is the recommended way.